demosplan/DemosPlanCoreBundle/ValueObject/APIPagination.php

```diff
@@ -22,7 +24,7 @@
  * @method self   setSortBy(string $sortBy)
  * @method self   setSortDirection(string $sortDirection)
  */
-class APIPagination extends ValueObject
+class APIPagination extends ValueObject implements ApiPaginationInterface
```

Comment:
The methods defined in the docblock above are now mostly defined in the ApiPaginationInterface and therefore mus be implemented in this class, otherwise the class needs to be declared abstract, which doesn't suit for ValueObjects afaik

Reply:
Bummer. Done in ba849603faf490e6fa73b6729042ac0146449ad7.